evm/logic/context.py

```diff
@@ -106,6 +106,10 @@ def gasprice(computation):
     computation.stack.push(computation.transaction_context.gas_price)
 
 
+def sharding_gasprice(computation):
```

Comment:
What do you think about calling this `PAYGAS_gasprice` so that it's clealy linked to the `PAYGAS` opcode.  The `sharding_` prefix doesn't feel like the correct descriptor.